ocaml_emulator/softfloat.ml

```diff
@@ -35,7 +35,11 @@ let f64_mul rm v1 v2 =
 let f64_div rm v1 v2 =
   ()
 
+<<<<<<< HEAD
```

Comment:
Incomplete merge process !

Reply:
Thanks！I will handle the conflicts I didn't find before carefully.